tests/Swagger/Serializer/DocumentationNormalizerTest.php

```diff
@@ -1229,14 +1229,21 @@ public function testFilters()
                 'required' => false,
                 'strategy' => 'partial',
             ]]),
+            'f3' => new DummyFilter(['toto' => [
+                'property' => 'name',
+                'type' => 'array',
+                'is_collection' => true,
+                'required' => true,
+                'strategy' => 'exact',
+            ]]),
         ];
 
         foreach ($filters as $filterId => $filter) {
             $filterLocatorProphecy->has($filterId)->willReturn(true)->shouldBeCalled();
             $filterLocatorProphecy->get($filterId)->willReturn($filter)->shouldBeCalled();
         }
 
-        $filterLocatorProphecy->has('f3')->willReturn(false)->shouldBeCalled();
+        $filterLocatorProphecy->has('f4')->willReturn(false)->shouldBeCalled();
```

Comment:
Not a fault of this PR, but `->shouldBeCalled()` should only be used when we actually want to check that a method is called. If the intention is only to mock the return value of the method, `->willReturn(...)` should suffice.